nginx/entrypoint.sh

```diff
@@ -92,6 +93,7 @@ events {
 pid         /var/run/nginx.pid;
 
 http {
+  server_names_hash_bucket_size  $SERVER_NAMES_HASH_SIZE;
```

Comment:
This looks like it's missing an escaped `$`

I think it needs to be like: `server_names_hash_bucket_size  \$SERVER_NAMES_HASH_SIZE;`

Reply:
Well, it seems to work on my branch without the `\` ... so I would leave it this way, no ?